admin/worker_4tests.m.template

```diff
@@ -86,8 +86,19 @@ if DO_LOGGING
 end
 
 keep_worker_running = true;
+num_of_runs = 0;
 while keep_worker_running
     mess_cache.instance().clear()
+    if DO_LOGGING
+        num_of_runs = num_of_runs+1;
+        fprintf(fh,'   ***************************************\n');
+        fprintf(fh,'   ******   RUN N : %d LabN %d  : ********\n',num_of_runs,intercomm.labIndex);
+        fprintf(fh,'   ****************************************=\n');
+        fprintf('   ***************************************\n');
+        fprintf('   ******   RUN N : %d LabN %d  : ********\n',num_of_runs,intercomm.labIndex);
+        fprintf('   ****************************************=\n');
+        
+    end
     %
     %----------------------------------------------------------------------
     % 2) step 2 of the worker initialization.
```

Comment:
can this file be renamed `worker_for_tests.m.template` -- we should not be using numbers are words in function, class or filenames.

Reply:
As we've discussed, too much hussle to do it at the moment. Left for backlog if you wish.